README.md

```diff
@@ -1 +1,21 @@
 # cargo_loading_service
+
+## Overview
+Cargo-Loading Service module gives control instructions to infrastructure.<br>
+There are three types of control instructions: process start, end, and error.
+
+## Input and Output
+- input
+  - from [in_parking_task_manager](https://github.com/tier4/in_parking_task_manager)
+    - `/in_parking/state` \[[tier4_v2x_msgs/msg/InfrastructureCommandArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/InfrastructureCommandArray.msg)\]:<br>State at vehicle stop points.
+  - from [v2i_interface](https://github.com/eve-autonomy/cargo_loading_service)
+    - `/v2i/infrastructer_states` \[[v2i_interface_msgs/msg/InfrastructureStateArray.msg](https://github.com/eve-autonomy/v2i_interface_msgs/blob/main/msg/InfrastructureState.msg)\]:<br>ROS2 interface from `v2i_status` (UDP).
+- output
+  - to [v2i_interface](https://github.com/eve-autonomy/v2i_interface)
+    - `/cargo_loding/infurastructre_commands` \[[tier4_v2x_msgs/msg/InfrastructureCommandArray](https://github.com/tier4/tier4_autoware_msgs/blob/tier4/universe/tier4_v2x_msgs/msg/InfrastructureCommandArray.msg)\]:<br>Control command to V2I infrastructure. It has an array structure to control multiple infrastructures at the same time.
+
```

Comment:
The user sees the list of interfaces here, so any omissions will confuse them.
Don't forget to include [service]/in_parking/task as well.

Reply:
fixed at https://github.com/eve-autonomy/cargo_loading_service/pull/17/commits/48015db6db5160485b3922435a0e4d77b9834ad2